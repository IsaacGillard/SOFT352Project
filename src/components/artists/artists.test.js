import React from 'react';
import { configure } from 'enzyme';
import { shallow } from 'enzyme';
import ArtistsPage from './ArtistsPage';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

test('ArtistsPage component should render as expected', () => {
  const component = shallow(<ArtistsPage />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
