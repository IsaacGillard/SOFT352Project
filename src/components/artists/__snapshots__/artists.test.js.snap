// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistsPage component should render as expected 1`] = `
<div>
  <div
    className="row justify-content-center"
  >
    <div
      className="col-10 col-sm-7 col-md-5 col-lg-4"
    >
      <AvForm
        method="get"
        model={Object {}}
        onInvalidSubmit={[Function]}
        onKeyDown={[Function]}
        onSubmit={[Function]}
        onValidSubmit={[Function]}
        tag={[Function]}
        validationEvent={
          Array [
            "onChange",
            "onInput",
          ]
        }
      >
        <AvGroup>
          <h2>
            <Label
              for="search"
              tag="label"
            >
              Search Artists
            </Label>
          </h2>
          <p>
            Find artists you own and add them to your MusicList.
          </p>
          <AvInput
            falseValue={false}
            id="search"
            name="search"
            onChange={[Function]}
            placeholder="Lorde"
            required={true}
            tag={[Function]}
            trueValue={true}
            type="text"
            validate={Object {}}
            validationEvent=""
            value=""
          />
          <AvFeedback>
            Required
          </AvFeedback>
        </AvGroup>
        <Button
          color="primary"
          tag="button"
        >
          Search Artists
        </Button>
      </AvForm>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-12 col-sm-12"
    >
      <div
        className="row"
      >
        <div
          className="col-sm-12 col-lg-12"
        />
      </div>
    </div>
  </div>
</div>
`;
